src/pages/developers/index.tsx

```diff
@@ -176,6 +176,10 @@ const StyledCard = styled(Card)`
   box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
   margin: 1rem;
   padding: 1.5rem;
+  justify-content: space-between;
```

Comment:
This change is ok but we want that change to be applied to all the `Card`s that we are using across the site. This change has impact only on this `developers/index` page.

Could you apply this style to the `src/components/Card.tsx` component instead of doing it here?

Reply:
ok

Reply:
![sssssss](https://user-images.githubusercontent.com/56462749/193739657-e12784fe-0956-4c69-bcb1-5e18a958af24.png)

Here the h3 means the title of card. 
if there is one line title -> the card title height is lets say 30px
if there us two line title -> the card title height will increase to lets say 67px.

so what im doing here is, setting the title height minimum 67px, so that even one line title will have the height of 67px. 
therefor the spacing will be even.